README.md

```diff
@@ -1,19 +1,251 @@
-# hazelcast-hibernate5
+# Hibernate Second Level Cache
```

Comment:
@mesutcelik I would like to hear your thoughts on referring to JCache configuration here. May be a link to [this](https://github.com/enozcan/hibernate-jcache) or including the directives directly.

Reply:
I'd not repeat information and just refer to it. Btw, shouldn't we move it over from your account to some repository under the Hazelcast organization?

Reply:
I think we should mention jcache integration here by pointing differences. I also agree with @pivovarit  about moving jcache sample to hazelcast-guides.

Reply:
Sure, I will move the content to hazelcast-guides.

Reply:
Moved: https://github.com/hazelcast-guides/hazelcast-hibernate-jcache-l2c